src/composables/trade/useGnosis.ts

```diff
@@ -63,6 +73,32 @@ type Props = {
   slippageBufferRate: ComputedRef<number>;
 };
 
+const PRICE_QUOTE_TIMEOUT = 10000;
+
+const priceQuotesResolveLast = onlyResolvesLast(getPriceQuotes);
+const feeQuotesResolveLast = onlyResolvesLast(getFeeQuote);
```

Comment:
This is a handy package to cancel previous promises and only use the result of the last promise. (so when you type into the text input slowly it will only grab the last one)